src/headless/brower.ts

```diff
@@ -266,7 +284,49 @@ export class HeadlessBrowser {
         logWithTimestamp("MetaMask activated and ready");
     }
 
+    async activateFlowWallet() {
+        const pages = this.context?.pages();
+        if (pages && pages.length === 0) {
+            logWithTimestamp("No page found, creating new page");
+            this.extensionPage = await this.context?.newPage();
+        }
+        if (this.extensionPage === undefined && pages && pages.length > 0) {
+            this.extensionPage = pages[pages.length - 1];
+        }
+        logWithTimestamp("Flow Wallet about to import account");
+
+        if (!this.extensionPage) {
+            logWithTimestamp("Extension page not initialized");
+            throw new Error("Extension page not initialized");
+        }
+
+        // ensure all the env variables are set
+        if (
+            !FLOW_WALLET_MNEMONIC ||
+            !FLOW_WALLET_USERNAME ||
+            !FLOW_WALLET_PASSWORD ||
+            !FLOW_WALLET_ADDRESS
+        ) {
+            logWithTimestamp("Flow wallet mnemonic, username, password, and address are not set");
+            throw new Error("Flow wallet mnemonic, username, password, and address are not set");
+        }
+
+        await importAccountBySeedPhrase({
```

Comment:
Stuck at here.
After disabling Headless, I found the UI gets stuck on the import page.